### DISCUSSION TITLE
What am I supposed to type here?

### DISCUSSION THREAD

**Discussion by 12122234** [FabricMC/fabric-api#2512] [Mod Dev Support] [answered]
[fabric-example-mod-1.19.zip](https://github.com/FabricMC/fabric/files/9507170/fabric-example-mod-1.19.zip)
![Screenshot (3)](https://user-images.githubusercontent.com/112907953/188908135-1b240167-0fcd-48b6-ba0f-ee523001464d.png)

**Comment by warjort** [comment, marked as answer]
Don't post text as images. Show your full class as text.
And your attached .zip file is empty so we can't see the important information there either.

Most likely you imported the wrong Registry class.
You should have:
````import net.minecraft.util.registry.Registry;````

**Reply by 12122234** [reply]
Sorry about the zip file being empty I didn't realize that, and your answer is right I did import the wrong Registry class.
